src/xcsync/SyncContext.cs

```diff
@@ -156,7 +156,12 @@ async Task SyncToXcodeAsync (CancellationToken token)
 		var appleDirectory = FileSystem.Path.Exists (altPath) ? altPath : FileSystem.Path.Combine (".", FileSystem.Path.GetDirectoryName (ProjectPath) ?? string.Empty);
 
 		foreach (var file in appleFiles) {
-			FileSystem.File.Copy (file, FileSystem.Path.Combine (TargetDir, FileSystem.Path.GetFileName (file)), true);
+			// get path relative to project, and then copy to target directory to ensure relative paths are maintained during sync
+			var relativePath = FileSystem.Path.GetRelativePath (FileSystem.Path.GetDirectoryName (ProjectPath)!, file);
```

Comment:
Does this show up correctly in Xcode, I don't see where you use the relative path in the  pbxproj content.

Reply:
![Screenshot 2024-10-04 at 11 32 24 AM](https://github.com/user-attachments/assets/628e803b-5420-4dc8-aefb-d1abcac53438)

it does now 😀